pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -56,29 +57,46 @@ var (
 // CreateCustomResourceDefinitions creates our CRD objects in the kubernetes
 // cluster
 func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) error {
-	if err := createCNPCRD(clientset); err != nil {
-		return err
-	}
+	g, _ := errgroup.WithContext(context.Background())
 
-	if err := createCCNPCRD(clientset); err != nil {
-		return err
-	}
+	g.Go(func() error {
+		if err := createCNPCRD(clientset); err != nil {
+			return err
+		}
+		return nil
```

Comment:
Nit: this could be shortened to:

```suggestion
		return createCNPCRD(clientset)
```

Same for the other changes below.